public/static/locales/fi/events.json

```diff
@@ -27,7 +27,7 @@
     "labelPlaces": "Paikat",
     "labelCategories": "Kategoriat",
     "labelTargetGroups": "Kohderyhmät",
-    "labelOtherClassification": "Muu luokittelu",
+    "labelOtherClassification": "Toimintaa",
```

Comment:
perhaps you could ask for translations? I would use `"labelActivities": "Aktiviteetit"`,